rocAL/rocAL_pybind/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ endif()
 if(${BUILD_RALI_PYBIND})
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3")
 
-    link_directories(${ROCM_PATH}/rpp/lib ${CMAKE_INSTALL_PREFIX}/lib)
+    link_directories(${ROCM_PATH}/rpp/lib ${ROCM_PATH}/mivisionx/lib)
```

Comment:
Please keep the change from the master branch for this file. the CMAKE_INSTALL_PREFIX is correct here and shouldn't be changed to ROCM_PATH}/mivisionx/.

Reply:
@asalmanp: somehow the CMAKE_INSTALL_PREFIX didn't have the right path when I tested this out.
Can you please test rocal_pybind module install by following the steps on the readme?

Reply:
I also faced similar issue. Only if I use ${ROCM_PATH}/mivisionx/lib its able to fetch the libraries.

Reply:
@shobana-mcw @asalmanp & @rrawther the TOT has the `${ROCM_PATH}/mivisionx/lib` path to build and run tests for rocAL pybind. Issue #573 with support to build pybind will improve the solution. @rrawther is this PR ready to be merged?